<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugWidget</class>
 <widget class="QWidget" name="DebugWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SysTray-X Debug</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SysTray-X.qrc">
    <normaloff>:/files/icons/SysTray-X.png</normaloff>:/files/icons/SysTray-X.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="unreadMailTextLabel">
         <property name="text">
          <string>Unread mail:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="unreadMailLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="newMailTextLabel">
         <property name="text">
          <string>New mail:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="newMailLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>254</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="test1PushButton">
       <property name="text">
        <string>Test 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="test2PushButton">
       <property name="text">
        <string>Test 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="test3PushButton">
       <property name="text">
        <string>Test 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="test4PushButton">
       <property name="text">
        <string>Test 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SysTray-X.qrc"/>
 </resources>
 <connections/>
</ui>
